kubo-rpc/src/http/pubsub.rs

```diff
@@ -250,60 +246,80 @@ mod tests {
         #[async_trait]
         impl IpfsDep for TestDeps {
             /// Get information about the local peer.
-            async fn lookup_local(&self) -> Result<PeerInfo, Error> {
+            async fn lookup_local(&self) -> Result<crate::PeerInfo, Error> {
```

Comment:
Does it make sense to make a use for these, so the changes are less?

`use crate::{Bytes, Cid, Error, GossipsubEvent, MultiAddr, PeerId, PeerInfo};`

Reply:
Yeah I'll do that.